ros/lib/cw_logging.py

```diff
@@ -0,0 +1,54 @@
+"""
+This module helps with streaming app logs to AWS CloudWatch.
+The logs are then sent to Kibana through an in-place AWS Lambda function.
+Reference: https://consoledot.pages.redhat.com/docs/dev/platform-documentation/tools/logging.html
+"""
+
+import watchtower
+from boto3.session import Session
+from botocore.exceptions import ClientError
+
+from ros.lib.config import (
+    get_logger,
+    CW_ENABLED,
+    AWS_ACCESS_KEY_ID,
+    AWS_SECRET_ACCESS_KEY,
+    AWS_REGION_NAME,
+    AWS_LOG_GROUP,
+)
+
+module_prefix = 'CLOUDWATCH LOGGING'
+
+
+def commence_cw_log_streaming(stream_name):
+    logger = get_logger(__name__)
+    key_id = AWS_ACCESS_KEY_ID
```

Comment:
@saltgen - Python does not actually prevent reassignment. Can we avoid it and use constant variables as they are instead of reassigning them? What do you think?

Reply:
@kgaikwad Checking on this and the tests

Reply:
@kgaikwad As  discussed, env vars are now being used directly and tests should be fine now
Changes :rocket: